Update tmr-meeting-recording-change.md (#5967)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/tmr-meeting-recording-change.md

## Set up the meeting recording option for OneDrive for Business and SharePoint

1. Install the Teams PowerShell module.

> [!NOTE]

> > Skype for Business Online Connector is currently part of latest Teams PowerShell Module.

Reviewer: ```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

> > If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector

Reviewer: ```suggestion
> If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
```

  a. Install [Teams PowerShell](https://docs.microsoft.com/microsoftteams/teams-powershell-install).

2. Launch PowerShell as an admin

> 3. Import the MicrosoftTeams and log in as a Teams admin.

Reviewer: ```suggestion
3. Import Microsoft Teams and log in as a Teams admin.
```
